src/server/schemas/search_species.yaml

```diff
@@ -10,7 +10,7 @@ properties:
     title: Namespace
     enum: ['rdp_taxonomy', 'ncbi_taxonomy', 'gtdb', 'silva_taxonomy']
   ts:
-    type: integer
+    type: [integer, "null"]
```

Comment:
Since `'ts'` already defaults to the current time if not provided, made sense to let `'ts'` be `None` to bypass the timestamp filter (which is allowable by the new stored query)